src/meta/src/hummock/manager/tests.rs

```diff
@@ -1695,12 +1694,13 @@ async fn test_split_compaction_group_trivial_expired() {
         .unwrap();
     let mut selector: Box<dyn CompactionSelector> =
         Box::<SpaceReclaimCompactionSelector>::default();
-    let reclaim_task = hummock_manager
-        .get_compact_task_impl(2, &mut selector)
+    let normal_tasks = hummock_manager
+        .get_compact_tasks_impl(vec![2], &mut selector)
         .await
-        .unwrap()
         .unwrap();
-    assert!(CompactStatus::is_trivial_reclaim(&reclaim_task));
+    assert!(normal_tasks.is_empty());
+    // let reclaim_task = normal_tasks.pop().unwrap();
+    // assert!(CompactStatus::is_trivial_reclaim(&reclaim_task));
```

Comment:
Please restore the comments.

Reply:
No.... After refactor, we can not test whether a task is trivial move ...

Reply:
Don't understand why, 🤔. Can we test if one particular task in a group is trivial move?

Reply:
We restore it now (